src/test/java/string_adder/StringCalculatorTest.java

```diff
@@ -19,14 +21,11 @@ void setUp() {
 		stringCalculator = new StringCalculator();
 	}
 
-	@Test
-	void sum_null() {
-		assertThat(stringCalculator.sum(null)).isEqualTo(0);
-	}
-
-	@Test
-	void sum_빈문자열() {
-		assertThat(stringCalculator.sum("")).isEqualTo(0);
+	@ParameterizedTest
+	@NullSource
+	@EmptySource
```

Comment:
두 개의 어노테이션을 합친 어노테이션도 제공합니다.
```suggestion
	@NullAndEmptySource
```